runtime/compiler/env/J9PersistentInfo.hpp

```diff
@@ -321,8 +320,9 @@ class PersistentInfo : public OMR::PersistentInfoConnector
    int32_t _countForRecompile;
 
 #if defined(J9VM_OPT_JITSERVER)
+   static JITServer::RemoteCompilationModes _remoteCompilationMode; // JITServer::NONE, JITServer::CLIENT, JITServer::SERVER
+
    JITServer::RemoteCompilationModes getRemoteCompilationMode() const { return _remoteCompilationMode; }
```

Comment:
I was under the impression that static methods can't be called on an instance so I left this as-is to avoid changing every call to this function. But apparently there is no such rule so I'll change it.